addons/l10n_ec/models/account_tax.py

```diff
@@ -51,3 +72,8 @@ def _get_tax_vals(self, company, tax_template_to_tax):
         string="Code ATS",
         help="Tax Identification Code for the Simplified Transactional Annex",
     )
+    l10n_ec_code_taxsupport = fields.Selection(
+        L10N_EC_TAXSUPPORTS,
+        string='Tax Support',
+        help='Indicates if the purchase invoice supports tax credit or cost or expenses, conforming table 5 of ATS'
+    )

```

Comment:
:warning: new fields cannot be added in stable

Reply:
:see_no_evil:

Reply:
Moved to enterprise in v16 (script TBD)